### PR TITLE
Add example for how to ignore for Review

### PR DESCRIPTION
**Pull request by EliMor**
# Description
A brief description of the changes.

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: https://circleci.com/docs/style/.

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

**Comment by EliMor** [comment]
Just for clarity. Hope it helps.

File: jekyll/_cci2/testing-orbs.md [hunk 1]
```diff
@@ -128,7 +128,14 @@ circleci local execute --job shellcheck/check
...
 
-The orb-tools orb includes a job `orb-tools/review` which will run a suite of tests against your orb designed to find opportunities to implement best practices and improve the quality of the orb. The "review" job was modeled closely after _ShellCheck_, and operates based on a list of rules called "RC" Review Checks. Each "RC" code corresponds to a specific rule, which can optionally be ignored.
+The orb-tools orb includes a job `orb-tools/review` which will run a suite of tests against your orb designed to find opportunities to implement best practices and improve the quality of the orb. The "review" job was modeled closely after _ShellCheck_, and operates based on a list of rules called "RC" Review Checks. Each "RC" code corresponds to a specific rule, which can optionally be ignored using the `exclude` parameter in your config.yaml file.
+
+Example:
+```yaml
+      - orb-tools/review:
+          filters: *filters
+          exclude: RC006,RC007
+```
```
Comment (rosieyohannan) [inline jekyll/_cci2/testing-orbs.md:138]:
```suggestion
```yaml
version: 2.1

orbs:
  orb-tools: circleci/orb-tools@11.1

workflows:
  my-workflow:
    jobs:
      - orb-tools/review:
          exclude: RC006,RC007
```
```

Comment (rosieyohannan) [reply jekyll/_cci2/testing-orbs.md:138]: @KyleTryon could you check the syntax here is correct?

**Comment by rosieyohannan** [comment]
@EliMor thank you so much for your contribution - just a suggested change to make the example more explicit about usage - and I've tagged an engineer to double check the syntax. :-D

**Comment by EliMor** [comment]
> @EliMor thank you so much for your contribution - just a suggested change to make the example more explicit about usage - and I've tagged an engineer to double check the syntax. :-D

no problem.

**Review by KyleTryon — APPROVED** [review]

File: jekyll/_cci2/testing-orbs.md [hunk 2]
```diff
@@ -128,7 +128,19 @@ circleci local execute --job shellcheck/check
...
+version: 2.1
+
+orbs:
+  orb-tools: circleci/orb-tools@11.1
+
+workflows:
+  my-workflow:
+    jobs:
+      - orb-tools/review:
+          exclude: RC006,RC007
```
Comment (rosieyohannan) [inline jekyll/_cci2/testing-orbs.md:143]: @EliMor just need to close the block with ``` and then good to go I think!

Comment (rosieyohannan) [inline jekyll/_cci2/testing-orbs.md:143]:
```suggestion
          exclude: RC006,RC007
```
```

**Comment by rosieyohannan** [comment]
Hello @EliMor! Would you mind just closing the code block in your example and then I can get this published? Thank you! :-D 🎉 🚀

**Review by Toooo23 — APPROVED** [review]

File: jekyll/_cci2/testing-orbs.md (hunk 2 above)
Comment (rosieyohannan) [inline jekyll/_cci2/testing-orbs.md:143]: 3
```suggestion
```yaml
version: 2.1

orbs:
  orb-tools: circleci/orb-tools@11.1

workflows:
  my-workflow:
    jobs:
      - orb-tools/review:
          exclude: RC006,RC007
```
```

**Review by rosieyohannan — APPROVED** [review]

**Pull request merged**